modules/core/src/errors/DeviceDisclosureError.ts

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
This error class is thrown if any errors happen when fetching the device disclosures.